### PR TITLE
sdk: Retry GCS API failures same as downloads

### PR DESCRIPTION
**Pull request by dm0-**
This should fix most build failures caused by GCS, since bad HTTP responses seem to be random incidents.

### CODE REVIEW

File: cmd/gangue/gangue.go [hunk 1]
```diff
@@ -126,13 +130,15 @@ func run(cmd *cobra.Command, args []string) {
 	}
 
 	// Download the file and verify it (unless disabled)
-	if verify {
-		err = sdk.UpdateSignedFile(output, source, client, gpgKeyFile)
-		if err == nil && !keepSig {
-			err = os.Remove(output + ".sig")
+	for i, err = 0, fmt.Errorf("dummy"); i <= retry && err != nil; i++ {
```
Comment (euank) [inline cmd/gangue/gangue.go]: Any reason not to use the `util.Retry` helper?

Comment (dm0-) [reply cmd/gangue/gangue.go]: I don't think its wait time is useful here.  The HTTP failure responses seem to happen one at a time, not through some period of downtime.

Comment (arithx) [reply cmd/gangue/gangue.go]: Would it not make sense to just use a duration of 0 in that case?

**Comment by dm0-** [comment]
Looking into it further, I think I'd prefer this approach.  I believe the backend failures originated at this API call, and retrying here also protects other commands (e.g. when `cork` downloads the SDK).

**Pull request merged**